src/components/auth/SettingsModal.tsx

```diff
@@ -116,13 +121,50 @@ export const SettingsMenuItem = React.memo(function SettingsMenuItem({ finalRef
 
     const onSave = useCallback(() => {
         setFavoriteCommander(commanderSelectValue);
+
         closeModal();
     }, [closeModal, commanderSelectValue, setFavoriteCommander]);
 
     function moxfieldLinkerToggle() {
         setShowMoxfieldLinker(!showMoxfieldLinker);
     }
 
+    function updateMoxfieldInputValue(event: any) {
+        setMoxfieldInputValue(event.target.value);
+    }
+
+    const MoxfieldProfileFinder = MoxfieldService.useGetMoxfieldProfile();
```

Comment:
let's move this to the top with the other hooks. Also, the general naming convetion I have been using is that the useNNN() should be assigned to a const called NNN. 

It kind of reads like english. "useGetMoxfieldProfile()" returns "getMoxfieldProfile". I'm personally a fan of having functions sound like actions.